data-es6.js

```diff
@@ -16662,6 +16665,7 @@ exports.tests = [
         return set instanceof S && set.has(123);
       */},
       res: {
+        closure:     true,
```

Comment:
That looks strange. Can you show me `closure` output for `class S extends Set {}`?